src/main/java/org/candlepin/subscriptions/event/EventController.java

```diff
@@ -78,9 +79,17 @@ public Stream<Event> fetchEventsInTimeRange(
   }
 
   public Stream<Event> fetchEventsInTimeRangeByServiceType(
-      String orgId, String serviceType, OffsetDateTime begin, OffsetDateTime end) {
+      String orgId,
+      String serviceType,
+      OffsetDateTime begin,
+      OffsetDateTime end,
+      OffsetDateTime processEndDateTime) {
```

Comment:
I think `asOfDateTime` might be a better name. You should add some comments around how this is used. Something like: "asOfDateTime is used to ignore events that are recorded outside a given hourly tally's window. Those events will be processed during the next hourly tally, and filtering them out here ensures they won't be processed twice."

Reply:
sounds good. Will add comments and change the variable name

Reply:
@kahowell have added the comment and changed the variable name.